### ISSUE TITLE
Icon document-properties is shifted

### ISSUE THREAD

**Issue by mueller-ma** [numixproject/numix-icon-theme#1325]
The "i" should be centered: https://github.com/numixproject/numix-icon-theme/blob/master/Numix/64/actions/document-properties.svg

Example KeePassXC: 
![keepassxc](https://user-images.githubusercontent.com/22525368/56285500-aa406600-6117-11e9-8666-ad0f52b55999.png)
Thunderbird menu: 
![thunderbird](https://user-images.githubusercontent.com/22525368/56285499-aa406600-6117-11e9-8d95-bf8aed69d30e.png)

**Comment by palob** [comment]
The issue is that there is a sheet underneath the i which isn't discernible from the background.

**Comment by Foggalong** [comment]
@mueller-ma Which GTK theme is this with?

**Comment by mueller-ma** [comment]
I'm using Cinnamon with the theme from Apricity OS with your icons. The icons provided by them should look quite familiar to you (https://github.com/Apricity-OS/apricity-icons/tree/master/Apricity%20Icons/apps/scalable).
https://github.com/Apricity-OS/apricity-themes-cinnamon
https://github.com/Apricity-OS/apricity-themes-gnome

**Comment by Foggalong** [comment]
Gotta love that they've forked the icon theme and then haven't updated t for _3 years_. The number of bug fixes that have occurred in that time is crazy. I'll message the devs see if we can't fix that.

In the mean time, @mueller-ma can you verify this happens with our master version of the theme?

**Comment by mueller-ma** [comment]
They shut down Apricity OS a few years ago: https://apricity-os.github.io/
I'm using an about one month old clone of https://github.com/numixproject/numix-icon-theme-circle and to fix the folder symlinks https://github.com/numixproject/numix-icon-theme-square.

**Comment by Foggalong** [comment]
Oh... is it just the GTK theme that you're using then? Might be worth checking whether the themes they were forked from have been updated since.

**Comment by mueller-ma** [comment]
I don't think the issue is related to a theme, as the icon is already shifted here: https://github.com/numixproject/numix-icon-theme/blob/master/Numix/64/actions/document-properties.svg
Using the default cinnamon theme the issue still exists.

**Comment by Foggalong** [comment]
Aye, it being shifted wasn't what I was talking about because that's the intended design of the icon. I was curious whether the updated theme used a different background colour to what we use for paper sheets (which is what's causing it to appear like there's a floating `i`).

The specific issue of the colours clashing with the GTK theme is something which will affect more than just this icon though. I'm not sure how we properly address that because (other than not using any greyscale background symbols) there'll always be some colour which causes this problem.

**Comment by palob** [comment]
Maybe a slight yellowish tint for the sheet?

**Comment by Foggalong** [comment]
Yeah, that could work if we do that for all pages. Will have to do some checks around to make sure it doesn't clash too much with the colours used by the most popular GTK themes (obvs not gonna be able to avoid that for all of the niche, more colourful ones).

**Comment by mueller-ma** [comment]
Could you add a small black border to the sheet?

**Comment by Foggalong** [comment]
@mueller-ma Borders like that generally wouldn't fit in - if you look across the various different types of icon in the Numix themes, very few of them have borders. If it's to be distinguished in any way it'll have to be through colour

**Comment by palob** [comment]
The sheet could have lines on it if it doesn't look too busy.

**Comment by Foggalong** [comment]
Lines could work, but I think it'd have to be in combination with something else otherwise the lines will similarly just be floating.

**Comment by palob** [comment]
Maybe we should use the grey from the mimetype icons here.
